LocalPackages/AppInfoRetriever/Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "AppInfoRetriever",
```

Comment:
Since I'm adding some code to retrieve information from 3rd party apps living in /Applications, I figured it would be good to modularize it into a Swift Package.